src/components/organisms/AppRouter/AppRouter.jsx

```diff
@@ -60,6 +61,11 @@ const AppRouter = () => {
         <Route path="/admin/venue/creation" component={VenueWizard} />
         <Route path="/admin/venue/edit/:venueId" component={VenueWizard} />
 
+        <Route
+          exact
+          path="/admin_v2/templates/:venueId"
+          component={AdminTemplates}
+        />
```

Comment:
Why are we setting `exact` on this route? The only case where that should matter for this route is if we're defining additional subroutes in this same `Switch` (not nested within another component), that need to be matched. And in that case, the better solution is to change the order so that the more specific routes are listed before the more generic routes, so that they match first

- https://reactrouter.com/web/api/Switch
- https://reactrouter.com/web/api/Route/exact-bool